libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -561,17 +561,17 @@ static CK_RV prvMbedTLS_Initialize( void )
 
         if( 0 != lMbedTLSResult )
         {
-            LogError( ( "Failed to seed the mbed TLS DRBG for the PKCS #11 module with error %s",
-                        mbedtlsHighLevelCodeOrDefault( lMbedTLSResult ) ) );
-            LogError( ( "%s", mbedtlsLowLevelCodeOrDefault( lMbedTLSResult ) ) );
+            LogError( ( "Could not initialize PKCS #11. Failed to seed the mbed TLS DRBG for the PKCS #11 module with mbed TLS error = %s : %s.",
```

Comment:
Log message could be shortened as 
```suggestion
            LogError( ( "Could not initialize PKCS #11. Failed to seed the mbed TLS DRBG: mbed TLS error = %s : %s.",
```